macros/utils/cross_db_utils/current_timestamp.sql

```diff
@@ -0,0 +1,63 @@
+{# 
+   Copyright 2021 dbt Labs, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. 
+#}
+
+{# 
+    dbt_utils.current_timestamp is using dbt_utils.type_timestamp which is going to be deprecated from dbt-utils version 0.9.0,
+    therefore dbt_utils.current_timestap is going to be deprecated as well (without having a similar macro at dbt-core).
+    We copied this macro, with a small change that supports the deprecation, so we could continue and use it in our project.
+#}
```

Comment:
@oravi do you think this is ok? we give them the copyright and explain why we had to duplicate.